src/SDFResultViewer.jl

```diff
@@ -12,6 +12,7 @@ using Unitful
 using PhysicalConstants.CODATA2018: c_0, ε_0, m_e, e
 # plots
 import Plots
+import PyPlot
```

Comment:
Also add it to the package environment. To do this, activate the package env than add the new dependency
```
]activate SDFResultViewer
add PyPlot
```

Reply:
Done. I just realized that Statistics and StatsBase might be needed too (for example, I use weights in photon_solid_angle_emission)